.gitignore

```diff
@@ -0,0 +1,3 @@
+consumer/src/vendor/*
```

Comment:
Do we really need `.idea` folder for this example? Let's remove it to keep it simple